caffe2/core/c10_operator.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+#include <vector>
+#include <ATen/core/dispatch/OpSchemaRegistration.h>
+#include <ATen/core/dispatch/KernelRegistration.h>
+#include <ATen/core/function_schema.h>
+
+namespace caffe2 {
+namespace detail {
+
+template<class Caffe2Operator> const c10::OperatorHandle& c10_op_handle_for_c2_op();
+template <class Caffe2Operator>
+void call_caffe2_op_from_c10(c10::Stack* stack, c10::KernelCache* cache) { // TODO Pass in correct cache type
+  // precondition: on the stack, there's an IValue for each input and an IValue for each output.
```

Comment:
Strictly speaking, the outputs are "inputs" right? And because the inputs are removed from the stack, that means the outputs have to get shifted around, is that right?

Reply:
In the JIT schema definition, they are inputs. From the view of the caffe2 operator, they are outputs.